omero/sysadmins/version-requirements.txt

```diff
@@ -180,23 +157,21 @@ Components
     or 8 (Oracle and OpenJDK), and there are some client issues with
     OpenJDK6.  Also note that Ice 3.6 drops Java 6 support.  For 5.2
     deprecate 7; this is because security support ends in April 2015.
-  * For MacOS X, separate application bundles for Java 6 and Java 7+
-    allow the use of Java 7 or 8 on all supported MacOS X versions.
   * The requirement for 7 in the Windows clients is less urgent, but
     will encourage migration to a non-deprecated Java version ahead of
     the removal of support for 6.
 
 * Apache
 
-  * [5.1] 2.2 supported, 2.4 recommended
   * [5.2] 2.2 deprecated, 2.4 recommended
+  * [5.3] 2.2 deprecated, 2.4 recommended
   * Rationale: 2.2 is the CentOS/RHEL 6.x and Ubuntu 12.04
     version which will be deprecated in 5.2.
```

Comment:
Sorry, I don't get what needs changing here--it reads correctly to me though I could be being totally blind!  I've made a tiny tweak to make it clearer.